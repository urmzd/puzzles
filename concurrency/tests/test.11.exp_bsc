Adding transaction: 12358 Cat Dov 150000 200
Block mined: 1 0 0x00000000 1
12358 Cat Dov 150000 200
0x000002a4 0x00b3f33c
