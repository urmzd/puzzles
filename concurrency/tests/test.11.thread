Thread 0
Thread 1
Thread 10
Thread 11
Thread 12
Thread 13
Thread 14
Thread 15
Thread 2
Thread 3
Thread 4
Thread 5
Thread 6
Thread 7
Thread 8
Thread 9
