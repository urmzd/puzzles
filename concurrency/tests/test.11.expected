Received event TRX with ID 11235
Adding transaction: 11235 Al_Alpha Beth_Beta 150000 40
Received event TRX with ID 11236
Adding transaction: 11236 Cat_Charlie Dov_Delta 150000 20
Received event TRX with ID 11237
Adding transaction: 11237 Eve_Echo Fin_Foxtrot 150000 25
Received event TRX with ID 11238
Adding transaction: 11238 Guy_Golf Hana_Hotel 150000 40
Received event TRX with ID 11239
Adding transaction: 11239 Iona_Indigo Jeff_Juliet 150000 20
Received event TRX with ID 11240
Adding transaction: 11240 Kit_Kilo Liam_Lima 150000 25
Received event TRX with ID 11241
Adding transaction: 11241 Mona_Mike Nick_November 150000 40
Received event TRX with ID 11242
Adding transaction: 11242 Otto_Oscar Peta_Papa 150000 20
Received event TRX with ID 11243
Adding transaction: 11243 Qu_Quebec Reya_Romeo 150000 25
Received event BLK with ID 1
Removing transaction: 11235 Al_Alpha Beth_Beta 150000 40
Removing transaction: 11238 Guy_Golf Hana_Hotel 150000 40
Removing transaction: 11241 Mona_Mike Nick_November 150000 40
Removing transaction: 11236 Cat_Charlie Dov_Delta 150000 20
Removing transaction: 11237 Eve_Echo Fin_Foxtrot 150000 25
Removing transaction: 11239 Iona_Indigo Jeff_Juliet 150000 20
Received event END
